blueoil/datasets/pascalvoc_2007.py

```diff
@@ -54,12 +54,12 @@ def num_max_boxes(self):
             return 42
 
     def _annotation_file_from_image_id(self, image_id):
-        annotation_file = os.path.join(self.annotations_dir, "{:06d}.xml".format(image_id))
+        annotation_file = os.path.join(self.annotations_dir, "{}.xml".format(image_id))
```

Comment:
Why `:06d` removed?
Related to pandas?

Reply:
> @lm-lily Can you remove pandas dependency from `setup.cfg`?
> https://github.com/blue-oil/blueoil/blob/master/setup.cfg#L26

@iizukak 
There is still delta-mark remained in Blueoil (issue #1066).
Is it ok to remove pandas dependency from `setup.cfg` before issue #1066 is solved?

Reply:
The file name extracted using my submitted code (after removing pandas dataframe method) failed with `:06d` format.
I have to adjust this part to make it right.

Reply:
> > The file name extracted using my submitted code (after removing pandas dataframe method) failed with :06d format.
> 
> Sorry, Why this happen?

The image_id has the format of sample data: ```2007_000032```. If we apply format ```{:06}``` to it, the filename will not match the format we need. It will be ```2007000032``` instead of ```2007_000032```, and hence cause the error of ```file not found```.

Reply:
@lm-lily I'm still not sure why this is related to removing pandas. If this is an existing bug, please make another PR to fix that.

Reply:
@tfujiwar 
Thank you Fujiwara san.
Then should I revert this change?

Reply:
@tfujiwar 
`lmnet-test` failed when I revert the above code.
```
[2020-07-20T05:41:28Z] self = <blueoil.datasets.pascalvoc_2007.Pascalvoc2007 object at 0x7ffb8908dc50>
--
  | [2020-07-20T05:41:28Z] image_id = '000085'
  | [2020-07-20T05:41:28Z]
  | [2020-07-20T05:41:28Z]     def _annotation_file_from_image_id(self, image_id):
  | [2020-07-20T05:41:28Z] >       annotation_file = os.path.join(self.annotations_dir, "{:06d}.xml".format(image_id))
  | [2020-07-20T05:41:28Z] E       ValueError: Unknown format code 'd' for object of type 'str'
  | [2020-07-20T05:41:28Z]
  | [2020-07-20T05:41:28Z] ../blueoil/datasets/pascalvoc_2007.py:57: ValueError

```
My previous explanation was a fault.
In the original code, the data is handled by pandas dataframe, which will handle the data type conversion internally.
My code takes in the file name as string type and that caused the lmnet-test failure.
Since the file name format is safe to be read as whole, I replace `{:06d}` to `{}` as the quick fix.
I will fix this code again and please advice if you have any better suggestion.